RecoMET/METFilters/plugins/EcalBadCalibFilter.cc

```diff
@@ -0,0 +1,171 @@
+// -*- C++ -*-
+//
+// Package:    ecalBadCalibSep2017ListFilter
+// Class:      ecalBadCalibSep2017ListFilter
+//
+/**\class ecalBadCalibSep2017ListFilter ecalBadCalibSep2017ListFilter.cc
+ 
+ Description: <one line class summary>
+ Event filtering to remove events with anomalous energy intercalibrations in specific ECAL channels
+*/
+//
+// Original Authors:  D. Petyt
+//
+ 
+ 
+// include files
+ 
+#include <iostream>
+ 
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+ 
+#include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/EcalDetId/interface/EEDetId.h"
+ 
+#include "Geometry/CaloTopology/interface/CaloTopology.h"
+#include "Geometry/CaloEventSetup/interface/CaloTopologyRecord.h"
+ 
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+ 
+using namespace std;
+ 
+ 
+class ecalBadCalibSep2017ListFilter : public edm::global::EDFilter<> {
```

Comment:
Class names should start with a capital letter.
Sorry if this was not clear in the previous iteration of review.

The part "Sep2017List" does not belong in the class name, it's relevant for specific configuration for an instance of this module. Please remove.